roles/installer/templates/deployments/task.yaml.j2

```diff
@@ -74,6 +74,28 @@ spec:
       priorityClassName: '{{ control_plane_priority_class }}'
 {% endif %}
       initContainers:
+{% if bundle_ca_crt  %}
+        - name: init-bundle-ca-trust
+          image: '{{ _init_container_image }}'
+          imagePullPolicy: '{{ image_pull_policy }}'
+          resources: {{ init_container_resource_requirements }}
+          securityContext:
+            runAsUser: 0
+            runAsGroup: 0
```

Comment:
@JoelKle This will break on Openshift. Can we template this in only if is_k8s=true?